CONTRIBUTING.md

```diff
@@ -355,11 +357,13 @@ After that you can run your test using the standard `pytest` commands. For examp
 pytest -s -vv tests/path-to-tests/test_file.py
 ```
 
-### Running system tests
+### Running `system_tests` (External contributors)
 
-To run a test in [tests/system_tests](tests/system_tests),
-install [Docker Desktop](https://www.docker.com/products/docker-desktop/) and launch a local test
-server:
+System tests will be automatically run when you create or update your GitHub pull request.
```

Comment:
I think this isn't true: someone with permissions has to trigger the tests. And it might not be possible for forks (I'm not sure though)

Reply:
yeah @timoffex is correct